libr/cons/cons.c

```diff
@@ -707,6 +707,7 @@ static bool lastMatters() {
 }
 
 R_API void r_cons_flush(void) {
+	static bool pageable = true;
```

Comment:
what about adding it as a variable to `RCons`? I'm not a big fan of static vars.

Reply:
yes, maybe useful for tasks because it swaps that global rconscontext struct, so it makes sense. changing